app/templates/album.html

```diff
@@ -1,24 +1,24 @@
 <main class="album-view container narrow">
   <section class="clearfix">
     <div class="column half">
-      <img src="{{ albumData.albumArtUrl }}">
+      <img src="{{ album.albumData.albumArtUrl }}">
```

Comment:
It's more like:

`controllerName.conrollerProperty.propertyProperty` if that makes sense.